pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -348,7 +348,9 @@ stdenv.mkDerivation ({
 
   passthru = passthru // {
 
-    inherit pname version;
+    inherit pname version ghc;
+
+    isHaskellPkg = true;
```

Comment:
I'd prefer a definition like `isHaskellPackage = drv: drv ? isHaskellLibrary` in `lib.nix`. That definition of `isHaskellPkg` seems redundant (and its name feels inconsistent with the spelled-out `isHaskellLibrary` rather than `isHaskellLib`).